renovate.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
this file exists in subscriptions as well, but i'm not sure we need it, as i think renovate is a JS thing? 🤔